homeassistant/components/media_player/kodi.py

```diff
@@ -480,8 +482,14 @@ def media_content_id(self):
 
     @property
     def media_content_type(self):
-        """Content type of current playing media."""
-        return MEDIA_TYPES.get(self._item.get('type'))
+        """Content type of current playing media.
+
+        If the media type cannot be detected, the player type is used.
+        """
+        if MEDIA_TYPES.get(self._item.get('type')) is None and self._players:
+            return MEDIA_TYPES.get(self._players[0]['type'])
+        else:
```

Comment:
Just in case it's not clear [travis is complaining about this else](https://travis-ci.org/home-assistant/home-assistant/jobs/353357562#L567). It's pointing out that we don't need the else here, we can just go right to the next return after the if block.